### PR TITLE
Unify event list view on organizer and event-series

### PR DESCRIPTION
**Pull request by wiffbi**
This PR tries to unify the event lists in orgaizer and event-series views. It converts all lists into panels similar to how lists are handled in customer account views – as `.full-width.alternating-rows`.

It also introduces a new panel element `.panel-subhead` which can contain additional information slighty offset through a light background-color (maybe at some point even bein toggled through a button).

~~All of the panel-creation code is also turned into a template tag.~~

Bildschirmfoto 2024-12-11 um 13 38 30
Bildschirmfoto 2024-12-11 um 13 39 17

### CODE REVIEW

**Comment by wiffbi** [comment]
I think we could/should update the panel-template-tag to support foldable details/summary panels as well. It makes it a bit more complicated as it needs to save state in the render-context for the other commands to output the right HTML-element (panelbody needs to close summary, not a div).

Calling it from the template it might work something like:

`{% panel "primary" foldable=True open=False %}`

Not sure this needs to be part of this PR though, but I’d love to have this template-tag. :)

**Comment by wiffbi** [comment]
The more I think about it, I will remove the template-tag from this PR. The panel-tag should support the new fieldset/legend accordion-panels as well, so I think this is worth having its own PR and time to think about how to best implement it.

**Comment by wiffbi** [comment]
With the new panel-look of the event-selection, I was able to remove the subevent-toggle and just use a details/summary element. It needs some if-else in the template to switch between the foldable and the always open panel, but once we have the panel-tag, that supports all different types of panels, it is just an additional parameter `foldable` to the panel-template-tag. For now, it is three more `if`s.

Bildschirmfoto 2024-12-12 um 09 50 14
Bildschirmfoto 2024-12-12 um 09 50 02
Bildschirmfoto 2024-12-12 um 09 48 03

**Review by raphaelm — APPROVED** [review]
For the view in an event series, the mobile view looks a little dense with the button touching the rest, does it make sense to move either the button or the status bubble to the right? Or adding some more space even if that requires more scrolling then?

image

On the organizer page, it's already a bit different:

image

File: src/pretix/presale/templates/pretixpresale/event/index.html [hunk 1]
```diff
@@ -70,32 +70,37 @@ <h2 class="content-header">
...
+        <details class="panel panel-{% if show_cart %}primary{% else %}default{% endif %}">
+            <summary class="panel-heading">
+    {% else %}
+        <div class="panel panel-default">
             <div class="panel-heading">
-                {% if subevent %}
-                    {% trans "Other dates" context "subevent" %}
+    {% endif %}
+                <h3 class="panel-title"><b>
+                {% if subevent and "date" not in request.GET and "filtered" not in request.GET %}
```
[the remark this answers is not in the document]
Comment (wiffbi) [reply src/pretix/presale/templates/pretixpresale/event/index.html:82]: Oh yes, you are right, that is the same! I will fix it first thing tomorrow.

**Comment by wiffbi** [comment]
Oh, my bad – they should look the same! I fixed the event-series view and added the clock to the organizer-view, so both lists look the same.

**Pull request merged**